configure

```diff
@@ -201,7 +202,7 @@ function install_pkgin
 {
 	local pkglist
 
-	pkglist="build-essential flex libxslt openjdk11 nodejs"
+	pkglist="build-essential flex libxslt openjdk11 nodejs pwgen"
```

Comment:
I have a doubt, the list of packages is pinned to the specific versions provided by the platform image pkgsrc installed, right?. I'm assuming that the answer is yes.